src/template.js

```diff
@@ -710,80 +710,254 @@ module.exports = {
+                    cond: ['and',
+                        ['==', '$.answers.p-mainapplicant-confirmation-method.q-mainapplicant-confirmation-method', 'email'],
+                        ['|role.all', 'mainapplicant', 'adult', 'incapable']
+                    ],
+                    data: {
+                        templateId: '80843f77-a68c-4d7a-b3c9-42fd0de271c2',
+                        emailAddress:
+                            '||/answers/p-mainapplicant-confirmation-method/q-mainapplicant-enter-your-email-address||',
+                        personalisation: {
+                            caseReference: '||/answers/system/case-reference||'
+                        },
+                        reference: null
+                    }
+                },
+                {
+                    description: 'Confirmation email - mainapplicant.applicant:child',
+                    type: 'sendEmail',
+                    // prettier-ignore
+                    cond: ['and',
+                        ['==', '$.answers.p-mainapplicant-confirmation-method.q-mainapplicant-confirmation-method', 'email'],
+                        ['|role.all', 'mainapplicant', 'child']
+                    ],
+                    data: {
+                        templateId: '668fac4a-3e1c-40e7-b7ac-090a410fbb03',
+                        emailAddress:
+                            '||/answers/p-mainapplicant-confirmation-method/q-mainapplicant-enter-your-email-address||',
+                        personalisation: {
+                            caseReference: '||/answers/system/case-reference||'
+                        },
+                        reference: null
+                    }
+                },
+                {
+                    description: 'Confirmation email - rep.applicant:adult:capable',
+                    type: 'sendEmail',
+                    // prettier-ignore
+                    cond: ['and',
+                        ['==', '$.answers.p-rep-confirmation-method.q-rep-confirmation-method', 'email'],
+                        ['|role.all', 'rep', 'adult', 'capable']
+                    ],
                     data: {
-                        templateId: 'l10nt:templateId{?lng,context,ns}',
-                        emailAddress: 'l10nt:emailAddress{?lng,context,ns}',
+                        templateId: 'b21f1aa7-cc16-41e7-8b8e-5c69e52f21f9',
+                        emailAddress:
+                            '||/answers/p-rep-confirmation-method/q-rep-enter-your-email-address||',
                         personalisation: {
                             caseReference: '||/answers/system/case-reference||'
                         },
-                        reference: null // required for notify api
+                        reference: null
                     }
                 },
-                sendSms: {
+                {
+                    description: 'Confirmation email - rep.mainapplicant.applicant:adult:incapable',
```

Comment:
should we add the `authority` label to this description to be explicit, and not infer it?

`Confirmation email - rep.mainapplicant:authority.applicant:adult:incapable`

Reply:
Will check with Julie on her return